src/main/markdown/release-notes.md

```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
 **Note** - M1 = first milestone, RC1 = first release candidate
 
 * * *
+<a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+
+### Highlights
+
+- Java 8 syntax supported. 
+
+- [JsInterop new specification](https://docs.google.com/document/d/10fmlEYIHcyead_4R1S5wKGs1t2I7Fnp_PaNaa7XTEk0) implemented.
+
+- GSS is no longer considered as experimental and old /CssResource is now deprecated.
+
+### Deprecations
+
+- Old CssResource syntax is now deprecated.
+
+### Compiler changes
+
+- Java 8 syntax is now supported by the compiler.
+- New optimizer framework that reruns the optimization process only on the part of the AST that were impacted by the changes since its last run. That speedup compilation of large application.
+-
+- Many bugfixes.
+
+### Library Changes
+- Generic Accessor for GWT Properties. You can now use `System.getProperty()` to access the value of any configuration or binding property at compile time.
+- [Configurable Checks](https://docs.google.com/document/d/1AegOijkqg9ix6wtMMPYU0JeDBZXIzSgopDifPV7gH9E/edit?usp=sharing)
+
+
+#### JDK emulation
+- Put operations with HashMap are 2.8x faster.
+- Long Division are 5x faster.
+- Double/Boolean are not boxed anymore.
+
+#### Logging and Stack Traces
+
+
+#### GSS support
+- GSS is not longer experimental but it's not enabled by default for easier and smoother migration to GWT 2.8. The `CssResource.enableGss` configuration property turns on GSS support. For more information on GSS migration, please refer to [this document](http://www.gwtproject.org/articles/gss_migration.html)
+- "For loop support" added
+- a new function concat() is now available to concatenate variable and string literal.
+- @gen-webkit-keyframes annotation: generate automaticaly @-webkit-keyframes block 
+
+#### Internationalization
+
+#### Browser API changes
+
+#### Widgets
+
+#### Dev Mode
+
+#### Super Dev Mode
+ - SDM caches on disk the MinimalRebuildCache object, which speeds up the first SDM compile on a restart of the process.
+
+#### Testing
+
```

Comment:
There will be some more items from with the latests submissions. Please don't forget to revisit.